scripts/dist

```diff
@@ -2,4 +2,4 @@
 
 rm -rf dist
 
-npx tsc --build --clean
+npx tsc
```

Comment:
why removing this?

Reply:
the --build flag does not seem to do anything different than just running tsc. The --clean flag just deletes the dist folder after compiling, so it is the same as running rm -rf dist. Again, let’s meet to discuss this.

Reply:
hope the misunderstanding is clear now, going with tsd --build --clean and the tsc --p ./tsconfig.build.json